GrabMigrator/Build.GrabMigrator.cs

```diff
@@ -1,14 +1,14 @@
 using Nuke.Common;
 
-partial class Build : NukeBuild
+internal partial class Build
 {
-    [Parameter("Grab-migrator config file")] readonly string GrabMigratorConfig;
-
-    Target GrabMigrator => _ => _
-     .Requires(() => GrabMigratorConfig)
-     .Executes(() =>
-     {
-         new GrabMigrator.GrabMigrator().Do(GrabMigratorConfig);
-     });
+    [Parameter("Grab-migrator config file")]
+    public static string GrabMigratorConfig { get; set; }
 
+    public Target GrabMigrator => _ => _
+        .Requires(() => GrabMigratorConfig)
+        .Executes(() =>
+        {
+            new GrabMigrator.GrabMigrator().Do(GrabMigratorConfig);
```

Comment:
In usings:
`using GrabMigrator;`
```suggestion
            var grabMigrator = new GrabMigrator();
            grabMigrator.Do(GrabMigratorConfig);
```

Reply:
It will lead to a compilation error:
error CS0118: 'GrabMigrator' is a namespace but is used like a type